server/context.go

```diff
@@ -351,20 +351,22 @@ func (ctx *Context) PGURL(user string) (*url.URL, error) {
 	} else {
 		options.Add("sslmode", "verify-full")
 		requiredFlags := []struct {
-			name  string
-			value *string
+			name     string
+			value    *string
+			flagName string
 		}{
-			{"sslcert", &ctx.SSLCert},
-			{"sslkey", &ctx.SSLCertKey},
-			{"sslrootcert", &ctx.SSLCA},
+			{"sslcert", &ctx.SSLCert, "cert"},
+			{"sslkey", &ctx.SSLCertKey, "key"},
+			{"sslrootcert", &ctx.SSLCA, "ca-cert"},
```

Comment:
@mberhault I just pushed #5645, which will fix the crash you were seeing. I would akin a flag not being in the map to a bug with us incorrectly handling a flag type, so I think the third option for a default value would be as detailed of an error message about the missing flag as possible (without crashing). This will allow us to quickly find and fix any `FlagMap` issues without them being swept under the rug.